src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius = 5;
```

Comment:
why 5? This is hardcode

Reply:
will it be right when I replace initialization to constructor and make it with random positive integer?

Reply:
Okay, I`ve understood